public/xmlns/content.xsd

```diff
@@ -118,6 +118,29 @@
     </xs:simpleType>
 
     <!-- element nodes -->
+    <!-- Paragraph - Vertical stack of content with margin/padding on the top and bottom -->
+    <xs:complexType name="paragraph">
+        <xs:choice minOccurs="1" maxOccurs="unbounded">
+            <xs:group ref="content:elements" />
+        </xs:choice>
+    </xs:complexType>
+
+    <!-- Tabbed content -->
+    <xs:complexType name="tabs">
+        <xs:sequence>
+            <xs:element name="tab" minOccurs="1" maxOccurs="unbounded">
```

Comment:
Didn't we mean to put a max on this? I know we said 2 as a "soft" max, but we probably should enforce a hard max at 3 or no more than 4.